js/script.js

```diff
@@ -994,6 +1006,9 @@ var validCombos = {
     
     "android,C++,ARM32,NNAPI":
         "Follow build instructions from <a href='https://aka.ms/build-ort-nnapi' target='_blank'>here</a>",
+
+    "android,C#,ARM32,NNAPI":
+        "Follow build instructions from <a href='https://aka.ms/build-ort-nnapi' target='_blank'>here</a>",
     
```

Comment:
[https://aka.ms/build-ort-android](https://aka.ms/build-ort-android) is invalid. Assuming it should be updated to be [https://onnxruntime.ai/docs/build/android.html](https://onnxruntime.ai/docs/build/android.html).

Is there a benefit to going via the aka.ms redirection? Seems like it might make it harder to pickup when changes in the gh_pages files affect a link. If I moved build/android.html I would search for links using that, but would be extremely unlikely to search for the aka.ms link.

Reply:
Some time ago we had some frequent docs updates and it was easier to update the redirects to fix vs fixing every link. Hopefully we don't continue changing the docs structure too much.